src/cljstyle/format/ns.clj

```diff
@@ -1,5 +1,6 @@
 (ns cljstyle.format.ns
   (:require
+    [cljstyle.format.indent :as indent]
```

Comment:
You shouldn't need to pull in the indent logic here; that's a different formatting rule and they shouldn't be coupled. Indentation composes with other rules by running _after_ the other logic has had a chance to rewrite things, insert line breaks, and so on.